outputs.tf

```diff
@@ -0,0 +1,2 @@
+# Empty Outputs to comply with https://www.terraform.io/docs/modules/index.html#standard-module-structure
+
```

Comment:
Thanks for adding this. Could we move the output declarations here?

Reply:
This should be all set, I've moved the outputs to this file